dbms/src/Storages/ColumnComment.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <string>
+#include <unordered_map>
+
+#include <Parsers/IAST.h>
+
+namespace DB
+{
+
+struct ColumnComment {
```

Comment:
Because comments are just strings, we should probably just store them as strings - this struct becomes unnecessary. A string can be easily converted to ASTIdentifier if necessary